.github/workflows/scripts.yaml

```diff
@@ -0,0 +1,34 @@
+name: scripts
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - ".github/workflows/scripts.yaml"
+      - "scripts/**"
+permissions:
+  contents: read
+
+concurrency:
+  group: '$${{ github.workflow }}-$${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        folder: [artifacthub, require-sync, validate, website]
+    steps:
+      - uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9 # v4.0.0
+        with:
+          go-version: '1.19'
+          cache: false
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@08e2f20817b15149a52b5b3ebe7de50aff2ba8c5 # v3.4.0
```

Comment:
do we want to copy .golangci-lint from GK so we have the same linters?

Reply:
Sure thing - Looks like the existing code will need some additional work for the larger set of linters.